static/js/message_edit.js

```diff
@@ -6,6 +6,11 @@ const currently_editing_messages = new Map();
 let currently_deleting_messages = [];
 const currently_echoing_messages = new Map();
 
+// constants defined to preserve choice of user
+// when moving topics.
```

Comment:
Changed to make this clear that these aren't constants!  These should likely be non-exported values without the all-caps name given the use case.

Reply:
I guess we can't de-export them, but we could rename them.